services-js/registry-certs/client/models/UploadableFile.test.ts

```diff
@@ -0,0 +1,100 @@
+import nock from 'nock';
+import { autorun } from 'mobx';
+
+import UploadableFile, { Status } from './UploadableFile';
+
+jest.mock('../queries/delete-birth-certificate-uploaded-file');
+
+import deleteBirthCertificateUploadedFile from '../queries/delete-birth-certificate-uploaded-file';
+const deleteBirthCertificateUploadedFileMock: jest.MockInstance<
+  ReturnType<typeof deleteBirthCertificateUploadedFile>,
+  Parameters<typeof deleteBirthCertificateUploadedFile>
+> = deleteBirthCertificateUploadedFile as any;
+
+const serverSuccessResponse = '{"filename":"sample.pdf","attachmentKey":"17"}';
```

Comment:
It would be good to have this defined as an object literal that you can put the `UploadResponse` type on.